angular/core/components/email_settings_page/email_settings_page_controller.coffee

```diff
@@ -1,11 +1,7 @@
-angular.module('loomioApp').controller 'EmailSettingsPageController', ($rootScope, $translate, Records, AbilityService, FormService, Session, $location, ModalService, ChangeVolumeForm) ->
-  $rootScope.$broadcast('currentComponent', {page: 'emailSettingsPage'})
+angular.module('loomioApp').controller 'EmailSettingsPageController', (Records, AbilityService, FormService, Session, $location, ModalService, ChangeVolumeForm, $rootScope) ->
```

Comment:
I'd like to try to enforce angular-provided services (those starting with a $) coming first in these lists